feature-libs/commerce-quotes/core/model/commerce-quotes.model.ts

```diff
@@ -0,0 +1,58 @@
+import { OrderEntry } from '@spartacus/cart/base/root';
+import { Price, PaginationModel, Principal } from '@spartacus/core';
+
+export interface Quote {
+  allowedActions?: string[];
+  cartId?: string;
+  code: string;
+  comments?: Comment[];
+  creationTime?: Date;
+  description?: string;
+  entries?: OrderEntry[];
```

Comment:
`should:` `OrderEntry` interface doesn't contain fields like `comments` and `configurationInfos` so it has to be  extended